app/assets/javascripts/components/user_profiles/instructor_stats.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// import ByStudents from './by_students'
+// import StudentStats from './student_stats.jsx';
+
+const InstructorStats = React.createClass({
+  render() {
+    return (
+      <div>
+        <h5> InstructorStats by </h5>
+        <div className= "user_stats">
+          <div className= "stat-display">
```

Comment:
At first sight, it looks like the `InstructorStats` and `StudentStats` component have similar UI. Why don't we create a functional component encompassing the `.user_stats` div?